### ISSUE TITLE
Using order by with unionAll generates mysql error

### ISSUE THREAD

**Issue by cameo99** [slick/slick#1738]
Slick 3.2.0, Scala 2.11.6, MySQL

```scala
val firstQuery = query
.filter(x => x.sensorId === sensorId)
.sortBy(_.timestamp.asc)
.take(1)

val lastQuery = query
.filter(x => x.sensorId === sensorId)
.sortBy(_.timestamp.desc)
.take(1)

firstQuery.unionAll(lastQuery).list
```
this generates

```mysql
select `ID` as x2, `SENSORID` as x3, `TIMESTAMP` as x4 from `SAMPLES` where `SENSORID` = '123' order by `TIMESTAMP` limit 1
union all 
select `ID` as x2, `SENSORID` as x3, `TIMESTAMP` as x4 from `SAMPLES` where `SENSORID` = '123' order by `TIMESTAMP` desc limit 1

```
which MySQL complains

> Incorrect usage of UNION and ORDER BY

Both select statements need to be wrapped in parenthesis in order to work properly.
This was working correctly in Slick 2.1 before I upgraded.

Not sure if this is already covered by this issue #1424

**Comment by chipz** [comment]
This bug is no longer happens, tried with Slick 3.2.3 and Scala 2.11.6

**Comment by d6y** [comment]
Thank you for reporting that this issue as resolved in 3.2.3 👍 I'll close this issue now.
